app/helpers/katello/concerns/smart_proxy_helper_extensions.rb

```diff
@@ -0,0 +1,28 @@
+module Katello
+  module Concerns
+    module SmartProxyHelperExtensions
+      def disks(storage)
+        mount_points = {}
+        mount_points.default = []
+        disks = []
+        storage.each do |name, values|
+          mount = values['mounted']
+          next if mount_points.keys.include?(mount)
+          mount_points[mount] << name
+          values['header'] = "#{mount_points[mount].to_sentence} (#{values['path']} on #{values['filesystem']})"
+          values['total'] = values.delete('1g-blocks')
+          disks << values
+
+
+        end
+        disks.group_by{|h| h['mounted']}.map{|_, hs| hs.reduce(:merge)}
+      end
+
+      def boolean_to_icon(boolean)
+        boolean = Foreman::Cast.to_bool(boolean)
+        boolean ?  icon_text('ok', '', {:kind => 'pficon'}) :
+        icon_text('error-circle-o', '', {:kind => 'pficon'})
```

Comment:
Splitting this across two lines without indentation is confusing to read. You could also consider shortening it to something like:

```
icon  = boolean ? 'ok' : 'error-circle-o'
icon_text(icon, '', {:kind => 'pficon'})
```